src/modules/users/users.controller.ts

```diff
@@ -85,4 +91,22 @@ export class UsersController {
       data: updatedUser,
     };
   }
+
+  @Put('/wishlist/:accommodationId')
```

Comment:
Move this endpoint to accomdation module.

Reply:
/properties/:propertyId/like

Reply:
but I'm adding accommodationId on user table, that is why i used it here